sdk/src/record-provider.ts

```diff
@@ -255,11 +314,11 @@ class NetworkRecordProvider implements RecordProvider {
      * const programManager = new ProgramManager("https://api.explorer.provable.com/v1", keyProvider, recordProvider);
      * programManager.transfer(1, "aleo166q6ww6688cug7qxwe7nhctjpymydwzy2h7rscfmatqmfwnjvggqcad0at", "public", 0.5);
      */
-    async findCreditsRecord(microcredits: number, unspent: boolean, nonces?: string[], searchParameters?: RecordSearchParams): Promise<RecordPlaintext> {
+    async findCreditsRecord(microcredits: number, searchParameters: RecordSearchParams,nonces?: string[]): Promise<OwnedRecord> {
```

Comment:
```suggestion
    async findCreditsRecord(microcredits: number, searchParameters: RecordSearchParams): Promise<OwnedRecord> {
```

I think we should probably put the nonces into the RecordSearchParams object for this. It means we'll have to change the Default RecordSearchParams impl a little, but that works.

Reply:
done